public/robots.txt

```diff
@@ -1,3 +1,2 @@
-# https://www.robotstxt.org/robotstxt.html
 User-agent: *
-Disallow:
+Disallow: /
```

Comment:
I would revert all of the changes here, as per https://github.com/sparkletown/sparkle/pull/1002#issuecomment-744134671

Reply:
Probably just model it after https://yoast.com/robots.txt and drop the `Disallow:` line entirely

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/1002/commits/d72d03a22db372c304a40f36435e1f7b045655c5